_articles/hosting/install-on-premise.md

```diff
@@ -52,16 +52,23 @@ This article will walk you through how to install and deploy Bitwarden to your o
        .\bitwarden.ps1 -restart
 6. Test your deployment. Visit the web vault at your configured domain name, register a new account, and log in.
 
-## System Requirements
+## Minimum System Requirements
 
 - Processor: x64, 1.4GHz or faster
 - Memory: 2GB of RAM or more
 - Storage: 10GB or more
 - Docker: Engine 1.8+ and Compose 1.17.1+
 
+## Recommended System Requirements
+
+- Processor: x64, 2GHz or faster
+- Memory: 16GB of RAM or more
+- Storage: 100GB Solid State or more
+- Docker: Engine 1.8+ and Compose 1.17.1+
+
 If you are looking for a quality provider with affordable prices, we recommend:
 
-[![Digital Ocean](/help/images/digital-ocean.png "Digital Ocean")](https://m.do.co/c/512986b01931){:target="_blank"}
```

Comment:
This will break when deployed. The way we have it set up it can't work in both environments. Revert

Reply:
reverted, but also awaiting final input from @sevensixseven for specs